src/load-files.tsx

```diff
@@ -3,20 +3,9 @@ import fs from 'fs'
 import glob from 'glob'
 import type {FileInfo} from './types'
 
-function loadFiles({
-  cwd = process.cwd(),
-  ignore = [
-    '**/__tests__/**',
-    '**/test/**',
-    '**/backend.+(js|ts|tsx)',
-    '**/setupTests.+(js|ts|tsx)',
-    '**/setupProxy.+(js|ts|tsx)',
-    '**/*.d.ts',
-  ],
-  ...rest
-} = {}): Array<FileInfo> {
+function loadFiles({cwd = process.cwd(), ...rest} = {}): Array<FileInfo> {
   const fileInfo = glob
-    .sync('src/**/*.+(js|html|jsx|ts|tsx|md|mdx)', {cwd, ignore, ...rest})
+    .sync('src/{exercise,final}/*.+(js|html|jsx|ts|tsx|md|mdx)', {cwd, ...rest})
```

Comment:
```suggestion
    .sync('src/{exercise,final,examples}/*.+(js|html|jsx|ts|tsx|md|mdx)', {cwd, ...rest})
```

Reply:
I was wondering what's the problem with the disabled `Commit suggestion` button here... fortunately you just updated the code and merged already 😂

Glad to hear you like the change, fingers crossed it will work in all workshops, not just in the example 🤞 feel free to tag me in a new issue if you discover some problem caused by this...

Reply:
Thanks! Will do 👍